README.md

```diff
@@ -284,23 +238,39 @@ sequenceDiagram
     participant O as Orchestrator
     participant R as Runner
     participant P as Processor
-    Note over O: Discovered Runners<br>and processors
-    loop For every runner
-        Note over R: Runner is started with cli
-        O-->>R: Startup with address and uri
-        R-->>O: RPC.Identify: with uri
+
+    Note over O: Initialize gRPC server on port 50051
```

Comment:
We probably want to make the port configurable, e.g., in case people already have something running on that port.

Reply:
do you think we should keep the sequence diagrams?
or could we just point to the proto readme?